src-plugins/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -106,13 +101,6 @@ itkFiltersToolBox::itkFiltersToolBox ( QWidget *parent ) : medFilteringAbstractT
 
     QObject::connect ( d->filters, SIGNAL ( currentIndexChanged ( int ) ), this, SLOT ( onFiltersActivated ( int ) ) );
 
-    QLabel * dataTypeLabel = new QLabel ( tr ( "Data type :" ) );
-    d->dataTypeValue = new QLabel ( tr ( "Unknown" ) );
-
-    QHBoxLayout * dataTypeLayout = new QHBoxLayout;
-    dataTypeLayout->addWidget ( dataTypeLabel );
-    dataTypeLayout->addWidget ( d->dataTypeValue );
```

Comment:
Exactly, we talked about that with @fcollot and it's in my TODO list to do (if we want) a dedicated toolbox about that. I wanted to talk about that in a close future to determine what kind of toolbox, where, etc with both of you.